components/uaepass-authenticator/pom.xml

```
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

```suggestion
  ~ Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Check all the places